wwr/about.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="blah blah bleh">
```

Comment:
Please add a proper description for your website. This description should explain the content of this page.

![image](https://github.com/user-attachments/assets/a59b1d6b-d135-47af-94d5-31e0b276b59f)